src/components/calcite-accordion/calcite-accordion.tsx

```diff
@@ -127,10 +127,8 @@ export class CalciteAccordion {
   }
 
   @Listen("calciteAccordionItemSelect") updateActiveItemOnChange(event: CustomEvent): void {
-    this.requestedAccordionItem = event.detail.requestedAccordionItem;
-    this.calciteAccordionChange.emit({
-      requestedAccordionItem: this.requestedAccordionItem
-    });
+    this.requestedAccordionItem = event.detail;
+    this.calciteAccordionChange.emit(this.requestedAccordionItem);
```

Comment:
Do we doc the detail? If so, I think it's better to have `detail` always be an object for consistency and future-proofing.

Reply:
No prob - preference on naming?

Reply:
A more useful detail from the `calcite-accordion` event might be something like:
```
{
  accordionMode: ---;
  activeItems: ---;
  selectedItem: --- ;
}
```
Since `activeItems` could be empty or contain multiple depending on the accordion's `selection-mode`, that would be reasonably useful to return here.

Maybe that would mean that the currently internal accordion item event could be public ?

Reply:
@jcfranco any thoughts on what a useful detail might be? We could leave as is pre-pr but "requestedAccordionItem" is more internal-event-sounding-y I think.